ui/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "babel-eslint": "^9.0.0",
```

Comment:
Is this needed? `babel-eslint` should be already there as a dependency of `react-scripts` and `eslint-plugin-patternfly-react`.

Reply:
probably not